Makefile

```diff
@@ -10,7 +10,7 @@
 # Naming convention:
 #	for stable releases we use "1.0.0" format
 #   for pre-releases, we use   "1.0.0-beta.2" format
-VERSION=4.3.5
+VERSION=4.3.6-dev.2
```

Comment:
do we want to merge dev tags here?
or should this be `4.3.6`?

Reply:
What about on `branch/*` we always cut alpha, beta, or final releases and cut dev tags on our own branch?

Reply:
I'm was trying to avoid a situation where I make a commit on a branch, cut a tag to test the release, then delete the branch and suddenly the tag is no longer valid. If tags are pointers to commits, though (which I think they are?) then the commit referenced in the tag should stick around once the branch is gone, right?